STL_Extension/include/CGAL/Concurrent_compact_container.h

```diff
@@ -233,6 +233,11 @@ class Concurrent_compact_container
     clear();
   }
 
+  bool is_used(const_iterator ptr) const
+  {
+    return (type(&*ptr)==USED);
+  }
+
```

Comment:
@cjamin Can you comment on this? That is a modification of `Concurrent_compact_container<..>`. Does not it need a synchronization primitive? What if another thread is modifying the status of `ptr`?